tests-wtr/dom-actions/utils.ts

```diff
@@ -0,0 +1,8 @@
+import {DomActions} from "../../lib/dom-actions";
+
+export const utils = {
```

Comment:
nothing new here, just moved to a reusable file

Reply:
why not export `instantiateDomActions()` directly? Enclosing it in another object seems unnecessary

Reply:
good point, changing